### DISCUSSION TITLE
Installation of quspin 0.3.7 via conda on a Mac M1 Max

### DISCUSSION THREAD

**Discussion by andreaselben** [QuSpin/QuSpin#605] [Q&A]
Hi, 

you state that in the new version 0.3.7, macbook ARM64 processors are now supported. Thank you, that's awesome! 

I am trying to install it on my Mac M1 Max via conda and have a question that may be very simple. When executing 

> conda install -c weinbe58 quspin

the solving of the environment fails and I obtain

> PackagesNotFoundError: The following packages are not available from current channels:
> 
>   - quspin
> 
> Current channels:
> 
>   - https://conda.anaconda.org/weinbe58/osx-arm64
>   - https://conda.anaconda.org/weinbe58/noarch
>   - https://conda.anaconda.org/pypi/osx-arm64
>   - https://conda.anaconda.org/pypi/noarch
>   - https://conda.anaconda.org/conda-forge/osx-arm64
>   - https://conda.anaconda.org/conda-forge/noarch
>   - https://repo.anaconda.com/pkgs/main/osx-arm64
>   - https://repo.anaconda.com/pkgs/main/noarch
>   - https://repo.anaconda.com/pkgs/r/osx-arm64
>   - https://repo.anaconda.com/pkgs/r/noarch

Why is this? Is my conda wrongly configured? 

I have python 3.10.8, cython 0.29.32, numpy 1.23.5 and have also install all the packages required for the manual installation (joblib six dill gmpy2 numexpr boost llvm-openmp numba openmpi-mpicc). Should I opt for the manual installation? 

Thank you and best wishes!

**Comment by mgbukov** [comment]
Currently, quspin for Mac M1 should be compiled manually (see documentation
website for instructions on how to do that).

We are working on releasing a precompiled version but this will take us
more time.

On Wed, Dec 14, 2022 at 11:55 PM Andreas Elben ***@***.***>
wrote:

> Hi,
>
> you state that in the new version 0.3.7, macbook ARM64 processors are now
> supported. Thank you, that's awesome!
>
> I am trying to install it on my Mac M1 Max via conda and have a question
> that may be very simple. When executing
>
> conda install -c weinbe58 quspin
>
> the solving of the environment fails and I obtain
>
> PackagesNotFoundError: The following packages are not available from
> current channels:
>
>    - quspin
>
> Current channels:
>
>    - https://conda.anaconda.org/weinbe58/osx-arm64
>    - https://conda.anaconda.org/weinbe58/noarch
>    - https://conda.anaconda.org/pypi/osx-arm64
>    - https://conda.anaconda.org/pypi/noarch
>    - https://conda.anaconda.org/conda-forge/osx-arm64
>    - https://conda.anaconda.org/conda-forge/noarch
>    - https://repo.anaconda.com/pkgs/main/osx-arm64
>    - https://repo.anaconda.com/pkgs/main/noarch
>    - https://repo.anaconda.com/pkgs/r/osx-arm64
>    - https://repo.anaconda.com/pkgs/r/noarch
>
> Why is this? Is my conda wrongly configured?
>
> I have python 3.10.8, cython 0.29.32, numpy 1.23.5 and have also install
> all the packages required for the manual installation (joblib six dill
> gmpy2 numexpr boost llvm-openmp numba openmpi-mpicc). Should I opt for the
> manual installation?
>
> Thank you and best wishes!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/weinbe58/QuSpin/discussions/605>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADHSUBQ6XWPJ3OC4UZEY7W3WNI64LANCNFSM6AAAAAAS67LTGA>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
Marin Bukov, PhD
Research Group Leader
Max Planck Institute for the Physics of Complex Systems
Nöthnitzer Str. 38
Dresden 01187, Germany

url: *https://www.pks.mpg.de/nqd <https://www.pks.mpg.de/nqd>*
<http://quantum-dynamics.phys.uni-sofia.bg/>
